Makefile

```diff
@@ -1,58 +1,37 @@
-TARGET_DIR=$(CURDIR)/_output
 KUBECONFIG?=$(HOME)/.kube/config
 
-GOBUILD=go build
-BUILD_GOPATH=$(TARGET_DIR):$(TARGET_DIR)/vendor:$(CURDIR)/cmd
+export GOBIN=$(CURDIR)/bin	   # Install built exes into bin/
+export PATH:=$(CURDIR)/bin:$(PATH) # Run executables from bin/ first.
 
-IMAGE_BUILDER_OPTS=
 IMAGE_BUILDER?=imagebuilder
-IMAGE_BUILD=$(IMAGE_BUILDER)
 export IMAGE_TAG_CMD?=docker tag
 
 export APP_NAME=cluster-logging-operator
-APP_REPO=github.com/openshift/$(APP_NAME)
-TARGET=$(TARGET_DIR)/bin/$(APP_NAME)
-IMAGE_TAG?=quay.io/openshift/origin-$(APP_NAME):latest
-export IMAGE_TAG
-MAIN_PKG=cmd/manager/main.go
-export OCP_VERSION?=$(shell basename $(shell find manifests/  -maxdepth 1  -not -name manifests -type d))
+export IMAGE_TAG?=quay.io/openshift/origin-$(APP_NAME):latest
+
+export OCP_VERSION?=4.5
 export CSV_FILE=$(CURDIR)/manifests/$(OCP_VERSION)/cluster-logging.v$(OCP_VERSION).0.clusterserviceversion.yaml
 export NAMESPACE?=openshift-logging
 export EO_CSV_FILE=$(CURDIR)/vendor/github.com/openshift/elasticsearch-operator/manifests/$(OCP_VERSION)/elasticsearch-operator.v$(OCP_VERSION).0.clusterserviceversion.yaml
```

Comment:
IIRC `CSV_FILE` and `EO_CSV_FILE` are unused.

Reply:
+1 removed.